.github/workflows/ci.yml

```diff
@@ -84,6 +85,8 @@ jobs:
   nuxt-playwright:
     name: Nuxt Playwright tests
     runs-on: ubuntu-latest
+    needs:
+      - build
```

Comment:
I wish it was possible to _use_ the build from this step for the Nuxt playwright. I thought about doing this before but hesitated because it means that the _start_ of the Nuxt playwright tests in CI is now _two_ build times instead of just one :confused: But I agree that this makes sense, especially to prevent the comment from showing up when it's just a build failure anyway.

Reply:
> I wish it was possible to use the build from this step for the Nuxt playwright.

Me too! I was thinking about that but doesn't seem like something trivial for now...

Reply:
We could push the build as an artifact or something :thinking: And then devise some way of skipping the build step in CI... though we need to wait talkback to be available or you run into a race condition, I think this is documented in the playwright configuration:

https://github.com/WordPress/openverse-frontend/blob/f56af03179a68b090067fe6f517bd4bafb6a888b/test/playwright/playwright.config.ts#L16-L21

This package might be a thing we could use with npm-run-all: https://www.npmjs.com/package/wait-on

Something like `npm-run-all -p talkback "wait-on localhost:3000" -s start`? There's some way of getting the right parallel and serial tasks into `npm-run-all` to make it work I think: https://www.npmjs.com/package/npm-run-all/v/1.8.0#run-a-mix-of-sequential-and-parallel-tasks